src/System.Device.Gpio/System/Device/Gpio/GpioDriver.cs

```diff
@@ -131,6 +131,31 @@ protected internal virtual ValueTask<WaitForEventResult> WaitForEventAsync(int p
     /// <param name="callback">Delegate that defines the structure for callbacks when a pin value changed event occurs.</param>
     protected internal abstract void RemoveCallbackForPinValueChangedEvent(int pinNumber, PinChangeEventHandler callback);
 
+    /// <summary>
+    /// Write the given pins with the given values.
+    /// </summary>
+    /// <param name="pinValuePairs">The pin/value pairs to write.</param>
+    public virtual void Write(ReadOnlySpan<PinValuePair> pinValuePairs)
```

Comment:
GpioDriver methods should be `protected internal`, as we don't want anybody to call these except trough the GPIO controller.